docs/CHANGELOG.asciidoc

```diff
@@ -99,6 +100,7 @@
   issue: {ml-issue}1136[#1136].)
 * Fix classification job failures when number of classes in configuration differs 
   from the number of classes present in the training data. (See {ml-pull}1144[#1144].)
+* Change instrumentation timestamp resolution to millisecionds. (See {ml-pull}1237[#1237].)
```

Comment:
typo: millisecionds -> milliseconds

Also, say it's data frame analytics, as the ML release notes for anomaly detection and data frame analytics all go in the same section.

Reply:
Updated